content/ember/v1/ember-reduce-computed-property.md

```diff
@@ -1,10 +1,12 @@
 ---
-id: ember-reducecomputed-ember-arraycomputed
+id: ember-reducecomputedproperty-ember-arraycomputedproperty
```

Comment:
Howdy folks, I don't know if we can actually change any of these ids at all. They are what is used to link to this particular deprecation from the ember-cli console.log deprecation warning. @locks can you confirm?

Reply:
sorry! I'll comment on the other PRs 😖 

I just saw it was closed and assumed it was merged!

Reply:
all good, sorry this was kind of a confusing hodgepodge. yeap let's discuss on https://github.com/ember-learn/deprecation-app/pull/87